website/docs/content/punctuation/punctuation.md

```diff
@@ -203,12 +195,14 @@ Use ⌥+' on MacOS to insert a curly apostrophe
 
 <DosDonts>
     <template #dont>
-        <p>☑️ Remind later.</p>
-        <p>Your competitor https://rival.com.</p>
+        <img src="static/period-dont.png" style="margin: auto" />
```

Comment:
There is something strange with this particular image. I always get this error when trying to `pnpm build` the website or just open the Punctuation page:
<img width="1150" alt="image" src="https://github.com/user-attachments/assets/3af7ede5-7d0d-451d-bd31-7125a722a153">

Reply:
Must be an issue in your local setup, it builds without errors on my system, and the cd/build job on github also runs successfully.